### PR TITLE
(refactor) Formatted with rustfmt

### PR DESCRIPTION
**Pull request by AndrewBrinker**
As the title says, this is just the current code, reformatted with rustfmt.

### CODE REVIEW

File: src/requested_path.rs [hunk 1]
```diff
@@ -43,12 +44,11 @@ impl RequestedPath {
         let index_path = self.path.join("index.html");
 
         match fs::metadata(&index_path) {
-            Ok(m) =>
-                if m.is_file() {
-                    Some(index_path)
-                } else {
-                    None
-                },
+            Ok(m) => if m.is_file() {
```
Comment (reem) [inline src/requested_path.rs]: how about instead rewriting this match as

```rust
Ok(m) if m.is_file() => Some(index_path),
  _ => None
```

File: src/static_handler.rs [hunk 2]
```diff
@@ -105,9 +103,10 @@ impl Handler for Static {
         if requested_path.should_redirect(&metadata, req) {
             // Perform an HTTP 301 Redirect.
             let mut redirect_path = match req.extensions.get::<OriginalUrl>() {
-                None => &req.url,
-                Some(original_url) => original_url,
-            }.clone();
+                                        None => &req.url,
+                                        Some(original_url) => original_url,
+                                    }
+                                    .clone();
```
Comment (reem) [inline src/static_handler.rs]: this is really weird

File: src/requested_path.rs (hunk 1 above)
Comment (AndrewBrinker) [inline src/requested_path.rs]: You're right, that's better.

File: src/static_handler.rs (hunk 2 above)
Comment (AndrewBrinker) [inline src/static_handler.rs]: Yeah. I'm not sure that I agree with rustfmt here. There may be a parameter to stop it from doing this, but I'm not seeing a clear choice. I'll change it back.

File: src/requested_path.rs [hunk 3]
```diff
@@ -43,12 +44,7 @@ impl RequestedPath {
         let index_path = self.path.join("index.html");
 
         match fs::metadata(&index_path) {
-            Ok(m) =>
-                if m.is_file() {
-                    Some(index_path)
-                } else {
-                    None
-                },
+            Ok(m) if m.is_file() => Some(index_path),
```
Comment (untitaker) [inline src/requested_path.rs]: rustfmt screwed this one up, the borrow checker complains about it.

Comment (reem) [inline src/requested_path.rs]: I believe this will work if you use `Ok(ref m)`. You can't move out of the target of a match in a pattern in a conditional.

**Comment by reem** [comment]
Since this has accumulated merge conflicts, I'll just run `cargo fmt` and make the changes.

**Pull request closed without merging**